### PR TITLE
Merge java and scala versions of logging.md

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/logging.md [hunk 1]
```diff
@@ -336,10 +379,10 @@ akka {
 One gotcha is that the timestamp is attributed in the event handler, not when actually doing the logging.
 
 The SLF4J logger selected for each log event is chosen based on the
-`Class[_]` of the log source specified when creating the
+@scala[`Class[_]`] @java[`Class`] of the log source specified when creating the
 `LoggingAdapter`, unless that was given directly as a string in which
-case that string is used (i.e. `LoggerFactory.getLogger(c: Class[_])` is used in
-the first case and `LoggerFactory.getLogger(s: String)` in the second).
+case that string is used (i.e. @scala[`LoggerFactory.getLogger(c: Class[_])`] @java[`LoggerFactory.getLogger(Class c)`] is used in
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/logging.md]: Hm, technically Class<?> I guess?

File: akka-docs/src/main/paradox/scala/logging.md [hunk 2]
```diff
@@ -350,10 +393,16 @@ shown below:
...
 @@@
 
-```scala
+Scala
+:   ```scala
 val log = Logging(system.eventStream, "my.nice.string")
 ```
 
+Java
+:   ```scala
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/logging.md]:
```
```java
```

File: akka-docs/src/main/paradox/scala/logging.md [hunk 3]
```diff
@@ -497,10 +562,10 @@ A more advanced (including most Akka added information) example pattern would be
 <a id="jul"></a>
 ## java.util.logging
 
-Akka includes a logger for [java.util.logging](https://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html#package.description).
+Akka includes a logger for [java.util.logging](https://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html#package.description)].
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/logging.md]: looks like a stray `]`?

**Review by ktoso — APPROVED** [review]
LGTM, only some small fixups

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Fixed the two typos I spotted, merging

**Comment by raboof** [comment]
:+1: thx!

**Pull request merged**